lib/features/add_property/view/add_features_view.dart

```diff
@@ -0,0 +1,213 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+const EdgeInsets _kScreenPadding =
```

Comment:
Why do u need this constant?

Reply:
@suesitran Basically, it's just a padding to be shared with multiple places, so i took it out